packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -182,7 +183,7 @@ const PersonalDetails = ({
                                             <div>
                                                 <Text size={isMobile() ? 'xs' : 'xxs'} align={isMobile() && 'center'}>
                                                     <Localize
-                                                        i18n_default_text={`Please remember that it is your responsibility to keep your answers accurate and up to date. You can update your personal details at any time in your ${showAccountSettingsLinkOrText()}.`}
+                                                        i18n_default_text={`Please remember that it is your responsibility to keep your answers accurate and up to date. You can update your personal details at any time in your ${account_settings_text}.`}
```

Comment:
you can use `values={{ account_settings_text }}`

Reply:
Ok Farrah